default/stringtables/en.txt

```diff
@@ -12245,13 +12264,19 @@ PLC_RACIAL_PURITY
 Racial Purity
 
 PLC_RACIAL_PURITY_DESC
-'''Provides benefits for single-species empires, but penalizes species other than that on the capital planet.
+'''The species on the empire capital planet asserts dominance over all other species in the empire. This provides benefits for single-species empires, especially if they are [[encyclopedia XENOPHOBIC_SPECIES_TITLE]], but penalizes any other species.
+
+Planets with the capital's species have various bonuses applied:
+• Increases [[metertype METER_TARGET_HAPPINESS]] by [[value PLC_RACIAL_PURITY_TARGET_STABILITY_SELF]], growing [[value PLC_RACIAL_PURITY_STABILITY_PER_TURN]] per turn.
```

Comment:
This is confusing. The effect (bonus that starts at 0 and grows to a maximum) is similar to indoctrination, place us a similar wording, too.